lib/iris/_representation/cube_summary.py

```diff
@@ -218,7 +218,18 @@ def __init__(self, title, attributes):
 class CellMethodSection(Section):
     def __init__(self, title, cell_methods):
         self.title = title
-        self.contents = [str(cm) for cm in cell_methods]
+        # self.contents = [str(cm) for cm in cell_methods]
```

Comment:
@trexfeathers Oops... you forgot to nuke this `wip` code comment :+1:

Reply:
d4843fc805e15c38f9510621c76136764aa8ef05